src/app/(payload)/hooks/beforeAudienceValidateHook.ts

```diff
@@ -0,0 +1,26 @@
+import { Resend } from 'resend'
+import { CollectionBeforeValidateHook } from 'payload'
+
+export const beforeAudienceValidateHook: CollectionBeforeValidateHook = async ({
+  data,
+  operation,
+}) => {
+  if (operation !== 'create' || typeof data?.name !== 'string') {
+    return data
+  }
+  console.log({ data, operation })
+  const { name } = data
+  try {
+    const resend = new Resend(process.env.RESEND_FULL_API_KEY || '')
+    const resp = await resend.audiences.create({ name })
+
+    if (resp.error) {
+      console.error(resp.error)
```

Comment:
if we mature this it would be great to log this error somewhere

Reply:
maybe we could use this library while we're small for error logging? 
https://axiom.co/pricing#pricing-table

Reply:
brief googl search suggests that it works well with next